cves/CVE-2017-12615.yml

```diff
@@ -118,9 +125,11 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
+  answer: The issue was reported responsibly to the Apache Tomcat Security team
```

Comment:
```suggestion
  answer: |
      The issue was reported responsibly to the Apache Tomcat Security team
```

Reply:
Need to add the pipe character to correctly use newlines in yml.

Reply:
Ah, I'm not to familiar with yml. So thanks for the advice.